omniauth-google-oauth2.gemspec

```diff
@@ -9,8 +9,8 @@ Gem::Specification.new do |gem|
   gem.name          = 'omniauth-google-oauth2'
   gem.version       = OmniAuth::GoogleOauth2::VERSION
   gem.license       = 'MIT'
-  gem.summary       = %(A Google OAuth2 strategy for OmniAuth 1.x)
-  gem.description   = %(A Google OAuth2 strategy for OmniAuth 1.x. This allows you to login to Google with your ruby app.)
+  gem.summary       = %(A Google OAuth2 strategy for OmniAuth 2.x)
```

Comment:
This should be fixed, not sure why all these PR's had the same incorrect change. =\

Reply:
Went ahead and made the correction. My original PR was based off of the other PR that didn't pass testing and I completely overlooked this more subtle mistake while focusing on the build breaking changes that were included.

Reply:
#426
> The PR I was building off of for reference